### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by errogaht**
Hello! There is russian translation.
And im add some hard coded not translated strings to all other languages.

### CODE REVIEW

File: resources/lang/fr/people.php [hunk 1]
```diff
@@ -107,6 +107,11 @@
     'reminders_next_expected_date' => 'le',
     'reminders_cta' => 'Ajouter un rappel',
     'reminders_description' => 'Nous vous enverrons un courriel pour chacun des rappels ci-dessous. Les rappels sont envoyés le matin du jour où l\'évènement se passe.',
+    'reminders_frequency' => 'every',
```
Comment (djaiss) [inline resources/lang/fr/people.php]: `chaque`

File: resources/lang/fr/people.php [hunk 2]
```diff
@@ -107,6 +107,11 @@
...
+    'reminders_date' => 'Date',
+    'reminders_content' => 'Content',
```
Comment (djaiss) [inline resources/lang/fr/people.php]: `Contenu`

File: resources/lang/fr/people.php [hunk 3]
```diff
@@ -107,6 +107,11 @@
...
+    'reminders_actions' => 'Actions',
+    'reminders_one_time' => 'One time',
```
Comment (djaiss) [inline resources/lang/fr/people.php]: `Unique`

**Comment by djaiss** [comment]
Also, #156 changes how languages are being used in the app. If you could stick to this that'd be awesome.

**Comment by errogaht** [comment]
ok, make all needed changes,

**Comment by djaiss** [comment]
❤️ thanks so much.

**Pull request merged**